js/functions.js

```diff
@@ -0,0 +1,39 @@
+// Функции для проверки длины строки.
+// Первая функция.
+
+function checkStringLength(string, maxLength) {
+  if (string.length <= maxLength) {
+    return true;
+  }
+  return false;
+}
+console.log(checkStringLength('Массив', 5));
+
+// Вторая функция (стрелочная).
+
+const checkLength = (string = '', maxLength = 1) => string.length <= maxLength;
+console.log(checkLength('Функционал', 10));
+
+// Функции для проверки. Является ли строка палиндромом?
+// Первая функция
+
+const isPalindrom = (string = '') => {
+  string = string.replaceAll(' ', '').toLowerCase();
+  let reverseString = '';
+  for (let i = string.length - 1; i >= 0; i--) {
+    reverseString += string[i];
+  }
+  return string === reverseString;
+};
```

Comment:
Форматировать код лучше так, циклы/условия/return отделяем пустой строкой от другого кода
```suggestion
const isPalindrom = (string = '') => {
  string = string.replaceAll(' ', '').toLowerCase();
  let reverseString = '';
  
  for (let i = string.length - 1; i >= 0; i--) {
    reverseString += string[i];
  }
  
  return string === reverseString;
};
```